UPGRADE-3.4.md

```diff
@@ -234,6 +234,27 @@ Translation
    and will be removed in 4.0, use `Symfony\Component\Translation\Writer\TranslationWriter::write` 
    instead. 
 
+ * Passing a `Symfony\Component\Translation\MessageSelector` to `Translator` has been
```

Comment:
Should probably be added to the component's changelog file too. And we also need to update the `UPGRADE-4.0.md` file accordingly.

Reply:
done